src/main/java/com/uber/cadence/internal/common/RpcRetryer.java

```diff
@@ -107,6 +114,10 @@ public Throwable getException() {
         });
   }
 
+  public static <T extends Throwable> void retryWithDefaultOption(RetryableProc<T> r) throws T {
```

Comment:
Maybe just use method overload with the same name `retry`? IMHO, this is less verbose and reads better.

Reply:
sure